### PR TITLE
fixes #18 refresh on save

### LINKED ISSUE #18: students table not refrshing on action
Whenever an action is performed like adding,editing and deleting the student table does not refresh.  
I would expect that after performing each action, the students should automatically update 

Screenshot from 2022-06-14 14-39-58

Screenshot from 2022-06-14 14-40-04

Screenshot from 2022-06-14 14-40-08

It shows successfully deleted but the student is still there
Screenshot from 2022-06-14 14-40-31

### PR DESCRIPTION
**Pull request by mulfranck**
Fixes #18 

A little shift of functions seems to do the trick
 - The Modal close when the save button is clicked.
 - The list updated

### CODE REVIEW

File: src/app/modules/reusable/student/save-student/save-student.component.ts [hunk 1]
```diff
@@ -72,6 +72,8 @@ export class SaveStudentComponent implements OnInit {
         error: (err) => addToMessageService(this.messageService, 'error', 'Error', `${err.message}`)
       })
     }
+
+    this.closeModal();
```
Comment (elroykanye) [inline src/app/modules/reusable/student/save-student/save-student.component.ts:76]: I personally think that this is not in the scope of this issue and should then not be included in your PR. Let's focus on ensuring that only what is necessary for a specific issue should be taken care of. Thanks ☺️

**Review by elroykanye — CHANGES_REQUESTED** [review]
Amazing! Some few comments below

File: src/app/modules/reusable/student/save-student/save-student.component.ts (hunk 1 above)
Comment (mulfranck) [reply src/app/modules/reusable/student/save-student/save-student.component.ts:76]: I am a bit confused as my understanding the `save-student` container controls the Modal for adding students, closing the modal is also controlled by the `save-student` components right? So to close the modal on __save__ the `closeModal` method should be called in the `saveStudent` method.

Comment (mulfranck) [reply src/app/modules/reusable/student/save-student/save-student.component.ts:76]: The _save-student_ container controls the Modal for adding students, closing the modal is also controlled by the _save-student_ components right? So to close the modal on __save__ the `closeModal` method should be called in the `saveStudent` method, so methought.

I must be misunderstanding something, do you have a suggestion as per this issue?

Comment (mulfranck) [reply src/app/modules/reusable/student/save-student/save-student.component.ts:76]: Okay, I get it now. Thanks.

**Comment by elroykanye** [comment]
I still think this could solve the issue except for the part where I stated is not necessary.

**Pull request closed without merging**